### ISSUE TITLE
[Question] grouped_gemm.ops.permute/unpermute is broken?

### ISSUE THREAD

**Issue by zzhhjjj** [tgale96/grouped_gemm#31]
I had an extremely large gradient norm during MoE training. I replaced them with the native PyTorch version, and the problem went away

**Comment by JayeeeZ** [comment]
Have you solved this problem? I found I can't use grouped_gemm.ops.permute …
